debug-on-kubernetes.md

```diff
@@ -0,0 +1,36 @@
+# Debug support on Kubernetes cluster
+
+## 1. Supported languages
+   * `java` (Required: [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension)
+   * `node`
+
+## 2. Commands for debugging
+   * `Kubernetes: Debug (Launch)` - Run the current application as a Kubernetes Deployment and attach a debugging session to it (currently works only for Java/Node.js deployments)
+   * `Kubernetes: Debug (Attach)` - Attach a debugging session to an existing Kubernetes Deployment (currently works only for Java deployments)
+
+## 3. How to use it for java debugging
+### 3.1 Launch a spring-boot application on Kubernetes and debug it

```

Comment:
Does 'spring-boot' need to be capitalised, or is all lower case idiomatic?

Reply:
Yeah, should be `Spring Boot`.